.github/workflows/lints.yml

```diff
@@ -36,11 +36,15 @@ jobs:
         id: nightly-toolchain
         run: echo "::set-output name=version::$(cat smart_contracts/rust-toolchain)"
 
+      - name: Get stable from rust-toolchain.toml
+        id: stable-toolchain
+        run: echo "::set-output name=version::$(grep -Eo '[0-9]+\.[0-9]+\.?[0-9]*' rust-toolchain.toml)"
```

Comment:
Could you consider using:
```
sed -nr 's/channel\s+=\s+\"(.*)\"/\1/p' rust-toolchain.toml
```

This way we'll also support toolchain specifiers such as
```
[toolchain]
channel = "stable"
```
and possibly all other combinations.

Ultimately, some toml parser could come in handy (something similar to `jq` for json).

Reply:
No worries, I was going with the recommendation in the ticket, but can do for sure yeah!